src/printapp/static/js/app.js

```diff
@@ -140,7 +140,38 @@ App.SignInController = Ember.Controller.extend({
 });
 
 App.QueueController = Ember.ArrayController.extend({
-  itemController: 'queueItem'
+  needs: ['application','queue'],
+  itemController: 'queueItem',
+
+  actions : {
+
+    deleteJob: function(job_id) {
+      $("#" + job_id + " .loading-spinner").css("display", "inline-block");
```

Comment:
I think jquery has a `.hide()` and `.show()` method which we should use instead.